docs/developer/base/basics.md

```diff
@@ -1,3 +1,169 @@
 # Basics
 
 -----
+
+The [AgentCheck base class](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/base.py) contains the logic that all Checks inherit.
+
+In addition to the integrations inheriting from AgentCheck, other classes that inherit from AgentCheck include:
+
+- [PDHBaseCheck](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/win/winpdh_base.py)
+- [OpenMetricsBaseCheck](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/openmetrics/base_check.py)
+- [KubeLeaderElectionBaseCheck](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/kube_leader/base_check.py)
+
+
+## Getting Started
+
+-----
+
+The Datadog Agent looks for `__version__` and a subclass of `AgentCheck` at the root of every Check package.
+
+Below is an example of the `__init__.py` file for a hypothetical `Awesome` Check:
+
+```python
+
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from .__about__ import __version__
+from .check import AwesomeCheck
+
+__all__ = ['__version__', 'AwesomeCheck']
+
+```
+
+Note that Datadog looks for `__version__` in `__about__.py`. This is the default location for all integrations, but it does not necessarily need to be put there. If no `__version__` is found, it will default to `0.0.0`.
```

Comment:
```suggestion
Note that the Agent looks for `__version__` in `__init__.py`. This is the default location for all integrations, but it does not necessarily need to be put there. If no `__version__` is found, it will default to `0.0.0`.
```